content/en/building/reference/app-settings/transitions.md

```diff
@@ -25,37 +25,27 @@ Transitions are run in series, not in parallel:
 
 Transitions obey the following rules:
 
-* has a `filter(doc)` function that accepts the changed document as an argument and returns `true` if it needs to run/be applied.
+* has a `filter({ id, doc, info })` function that accepts the changed document as an argument and returns `true` if it needs to run/be applied.
 
-* a `onMatch(change, db, auditDb, callback)` function that will run on changes that pass the filter.
```

Comment:
I searched for "audit" and this popped. Really ancient stuff. I OCD doctored it a bit so it's closer to current times. Someone imagined people will add their own transitions at the point when this documentation was written.

Reply:
😅